app/generic/issuer/facades/acapy_issuer_v1.py

```diff
@@ -0,0 +1,146 @@
+import logging
```

Comment:
genereally, but that's really personal style/preference, why make this a class/classes at all? Maybe functions is more lightweight/faster? You could access it like a class via import eg
```python
import .generic.issuer.facade.acapy_issuer_v1 as IssuerV1
```

then call the methods on that which are just functions. That saves instantiating all that dundar method stuff one gets for class objects.

really just as an idea. Totally fine if you wanna do it this way.